subprojects/docs/src/docs/userguide/javaLibraryPlugin.xml

```diff
@@ -0,0 +1,277 @@
+            It's a very common use case in multi-project builds, but also as soon as you have external dependencies: by definition, any
+            dependency is a Java library.
+        </para>
+        <para>
+            The plugin exposes two <firstterm>configurations</firstterm> that can be used to declare dependencies:
+            <literal>api</literal>
+            and <literal>implementation</literal>. The <literal>api</literal> configuration should be used to declare dependencies which
+            are exported by the library API, whereas the <literal>implementation</literal> configuration should be used to declare dependencies
+            which are internal to the component.
+        </para>
+        <sample id="useJavaLibraryPlugin" dir="java-library/quickstart" title="Declaring API and implementation dependencies">
+            <sourcefile file="build.gradle" snippet="dependencies"/>
+        </sample>
+        <para>Dependencies appearing in the <literal>api</literal> configurations will be transitively exposed to consumers of the library, and
+            as such will appear on the compile classpath of consumers. Dependencies found in the <literal>implementation</literal> configuration will,
+            on the other hand, not be exposed to consumers, and therefore not leak into their own compile classpath. This comes with several benefits:
+        </para>
+        <itemizedlist>
+            <listitem>dependencies do not leak into the compile classpath of consumers anymore, so you will never accidently depend on a transitive dependency</listitem>
+            <listitem>faster compilation thanks to reduced classpath size</listitem>
+            <listitem>less recompilations when implementation dependencies change: consumers would not need to be recompiled</listitem>
+            <listitem>cleaner publishing: when used in conjunction with the new <literal>maven-publish</literal> plugin, Java libraries produce POM files
+                representing exactly what is required to compile against the library, or use at runtime (in other words, don't mix what is needed to compile
+                the library itself and what is needed to compile against the library).
+            </listitem>
+        </itemizedlist>
+        <note>The <literal>compile</literal> configuration still exists but should not be used as it will not offer the guarantees that the
+            <literal>api</literal>
+            and <literal>implementation</literal> configurations provide.
+        </note>
+    </section>
+
+    <section id="sec:java_library_recognizing_dependencies">
+        <title>Recognizing API and implementation dependencies</title>
+        <para>As soon as we want to separate API and implementation dependencies, we reckon some guidelines as to make the difference between the two.
+            Basically, an API dependency is a type that is exposed in the library binary interface, often referred to ABI (Application Binary Interface).
```

Comment:
`...the library binary interface, often referred to ABI...` ->
`...the library's binary interface, often referred to as ABI...`